web/frontend/src/components/keyboard-input.vue

```diff
@@ -114,36 +114,35 @@ onUnmounted(() => {
 <template>
   <div
     ref="root"
-    class="h-23 flex w-full flex-col items-center"
   >
-    <div class="flex w-48 gap-2 pb-4">
-      <v-switch
-        :label="isActive ? 'Keyboard Enabled' : 'Keyboard Disabled'"
-        class="pr-4"
-        :value="isActive ? 'on' : 'off'"
-        @input="toggleKeyboard($event.target.value === 'on')"
-      />
-    </div>
-    <div
-      v-for="(lineKeys, index) in keysLayout"
-      :key="index"
-      class="my-1 flex flex-row justify-center gap-2"
-    >
-      <button
-        v-for="key in lineKeys"
-        :key="key"
-        class="flex items-center gap-1.5 border border-gray-500 px-3 py-1 outline-none"
-        :class="{
-          'bg-gray-200 text-gray-800': pressedKeys[key],
-          'bg-white': !pressedKeys[key],
-        }"
-        @pointerdown="handlePointerDown(key)"
-        @pointerup="handlePointerUp(key)"
-        @pointerleave="handlePointerUp(key)"
+    <v-switch
+      :label="isActive ? 'Keyboard Enabled' : 'Keyboard Disabled'"
+      class="pr-4"
+      :value="isActive ? 'on' : 'off'"
+      @input="toggleKeyboard($event.detail.value)"
+    />
+    <div class="flex gap-2">
+      <div
+        v-for="(lineKeys, index) in keysLayout"
+        :key="index"
+        class="my-1 flex flex-col self-end gap-2"
       >
-        {{ key.toUpperCase() }}
-        <Icon :path="keyIcons[key]" />
-      </button>
+        <button
+          v-for="key in lineKeys"
+          :key="key"
+          class="flex items-center gap-1.5 border border-gray-500 px-3 py-1 outline-none"
+          :class="{
+            'bg-gray-200 text-gray-800': pressedKeys[key],
+            'bg-white': !pressedKeys[key],
+          }"
+          @pointerdown="handlePointerDown(key)"
+          @pointerup="handlePointerUp(key)"
+          @pointerleave="handlePointerUp(key)"
+        >
+          {{ key.toUpperCase() }}
```

Comment:
should we use `text-transform: uppercase` instead?

Reply:
yes will do